ngi_reports/stockholm/project_summary.py

```diff
@@ -260,12 +365,14 @@ def get_accredit_info(self,keys):
             try:
                 accredit = self.proj_details['accredited_({})'.format(k)]
                 if accredit in ['Yes','No']:
-                    accredit_info[k] = "{} Validated under ISO accreditation 17025:2005".format(["[cross]","[tick]"][accredit == "Yes"])
+                    accredit_info[k] = "{} under ISO accreditation 17025:2005".format(["[cross] Not validated","[tick] Validated"][accredit == "Yes"])
                 else:
-                    accredit_info[k] = accredit
+                    accredit_info[k] = accredit.replace("N/A","Not Applicable")
```

Comment:
It will be `NA` when project co-ordinator chooses `NA` (for finished library projects and etc), it will be empty if project co-ordinator forgets to choose something (which he shouldn't).